Handler/Events.hs

```diff
@@ -73,6 +73,25 @@ addListMetaData keyValues = do
   return $ keyValues ++ metaData
 
 
+text2Order :: Text -> [SelectOpt Event]
+text2Order text =
+  case lookup textWithNoPrefix keyVal of
+    Just val -> [direction val]
+    Nothing -> error "wrong order"
+  where
+    keyVal = [ ("title", EventTitle)
```

Comment:
seems it doesn't like `keyVal` as the tuple is different. So there must be a better way :)

@dawehner in case you want to play around a bit with Haskell/Yesod :)